pom.xml

```diff
@@ -63,21 +61,11 @@
 
     <modules>
         <module>flyway-core</module>
-        <module>flyway-gradle-plugin</module>
         <module>flyway-maven-plugin</module>
         <module>flyway-commandline</module>
-
-
-
-
-
-
-
-
     </modules>
 
     <distributionManagement>
```

Comment:
You need to remove this distribution management stuff since we have this in our parent pom and we need to use our Nexus